influxd.yaml

```diff
@@ -61,6 +61,54 @@ subpackages:
         - bash
         - yq
 
+  - name: ${{package.name}}-${{package.version}}-bitnami-compat
+    description: "compat package with bitnami/influxdb image"
+    pipeline:
+      - uses: bitnami/compat
+        with:
+          image: influxdb
+          version-path: 2/debian-12
+      - runs: |
+          mkdir -p ${{targets.subpkgdir}}/opt/bitnami/influxd/bin
```

Comment:
we should also consider adding `chmod g+rwX /opt/bitnami` just to be cautious given this is a stateful application.

Reply:
```
chmod -R u+rwX,g+rwX,o+rw ${{targets.contextdir}}/opt/bitnami/
chmod -R u+rwX,g+rwX,o+rw ${{targets.contextdir}}/bitnami/influxd
```